tests/test_fields.py

```diff
@@ -47,6 +47,17 @@ def test_include_resource_linkage_single(self, post):
         assert result['data']
         assert result['data']['id'] == str(post.author.id)
 
+    def test_include_resource_linkage_single_with_schema(self, post):
+        field = Relationship(
+            related_url='/posts/{post_id}/author/',
+            related_url_kwargs={'post_id': '<id>'},
+            include_resource_linkage=True, type_='people', schema='PostSchema'
```

Comment:
Does this test fail without your change? `PostSchema` does not override `get_attribute`.